### ISSUE TITLE
Early warning: apple_static_library has been removed from Bazel HEAD

### ISSUE THREAD

**Issue by Wyverald** [bazelbuild/rules_apple#1512]
The built-in rule `apple_static_library` was removed in https://github.com/bazelbuild/bazel/commit/589354cdcc274edc01e5634c081739737c593f5e. It hasn't made it into a rolling release yet, but once it does, a lot of the tests in this repository won't work anymore (see for example https://buildkite.com/bazel/rules-apple-darwin/builds/5610#0181b728-d9f9-4a54-98e1-bc53bc705b9e).

**Comment by keith** [comment]
yep thanks we've seen. we'll have to come up with a hack as we still support bazel LTS
